cropduster/resizing.py

```diff
@@ -259,6 +259,7 @@ def create_image(self, output_filename, width, height):
             temp_file.write(f.read())
         temp_file.seek(0)
         image = PIL.Image.open(temp_filename)
+        image.name = self.image.filename
```

Comment:
typo? Should this be `image.filename`?

Reply:
Mm- unfortunately not. For individual thumbs, the filename ends up being a `default_storage` inaccessible tmp path - from [here](https://github.com/theatlantic/django-cropduster/blob/master/cropduster/resizing.py#L261).

Reply:
sorry, I meant, shouldn't this be:

```python
image.filename = self.image.filename
```

Reply:
If we did do that, we'd need to update the save method of the gifsicle object to 
```
with default_storage.open(self.pil_image.filename, 'rb') as f:
    out, err = proc.communicate(input=f.read())
```
But since we don't set the `filename` on pillow images on upload, just the name- see [me](https://github.com/theatlantic/django-cropduster/blob/master/cropduster/views/__init__.py#L236), it errors out.

Reply:
Ah. Then I think that is also a typo. grep for `.filename` in the code and you'll see we use that everywhere else, because that's what it's called in Pillow.

https://github.com/python-pillow/Pillow/blob/fe3e5809a344a63f7c801ee52a3dbc3a9706cf23/src/PIL/ImageFile.py#L96

Reply:
Updated 👍